pkg/api/uploads.go

```diff
@@ -3,8 +3,8 @@ package api
 import "time"
 
 type AddUploadsRequest struct {
-	Uploads   []Upload   `json:"uploads"`   // List of unfinished uploads
-	Artifacts []Artifact `json:"artifacts"` // List of created artifacts
+	Uploads   []Upload   `json:"uploads" validate:"required"`   // List of unfinished uploads
+	Artifacts []Artifact `json:"artifacts" validate:"required"` // List of created artifacts
```

Comment:
i believe artifacts also isn't required, just the uploads. there isn't a public api to finish the uploads (which creates the artifacts) so someone using only the public apis would only ever send upload uuids